js/plugins/google-cloud/src/gcpOpenTelemetry.ts

```diff
@@ -188,6 +189,24 @@ export class GcpOpenTelemetry implements TelemetryConfig {
       new PinoInstrumentation({ logHook: this.gcpTraceLogHook }),
     ];
   }
+
+  private buildMetricExporter(): PushMetricExporter {
+    const exporter: PushMetricExporter = this.shouldExportMetrics()
+      ? new MetricExporter({ projectId: this.options.projectId })
+      : new InMemoryMetricExporter(AggregationTemporality.DELTA);
+    exporter.selectAggregation = (instrumentType: InstrumentType) => {
+      if (instrumentType === InstrumentType.HISTOGRAM) {
+        return new ExponentialHistogramAggregation(100, true);
```

Comment:
nit: document these values inline

Reply:
Updated to use the defaults and not pass these values in at all.